src/kirby/components/modal/modal-wrapper/modal-wrapper.component.html

```diff
@@ -1,15 +1,17 @@
 <div class="modal-wrapper" #modalWrapper tabindex="-1" (window:focus)="onFocusChange()" (window:focusout)="onFocusChange()">
     <div class="modal-header">
-        <h2 class="text-{{config.titleHorizontalAlignment}} title">{{ config.title }}</h2>
-    </div>
+        <div class="title-container">
+            <span class="text-{{config.closeBtnPlacement === 'inside' ? 'left' : 'center'}} title">{{ config.title }}</span>
```

Comment:
I would move this to a method in the controller to avoid repeating the logic for web and {N}

Reply:
this logic has changed now, so i'll resolve this comment for now